src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -552,6 +553,7 @@ static RemoteSessionNamedPipeServer()
             IPCNamedPipeServerEnabled = true;
 
             CreateIPCNamedPipeServerSingleton();
+
 #if !CORECLR // There is only one AppDomain per application in CoreCLR, which would be the default
```

Comment:
This code attempts to dispose the default pipe server on process (appdomain) exit, but it is disabled for CORECLR and so is dead code.  This code should be removed or fixed to dispose both the default and custom pipe server objects.  Can you create a new Issue to look into it?

Reply:
Create the issue:
https://github.com/PowerShell/PowerShell/issues/8925